test/primitives/array-element-test.js

```diff
@@ -48,12 +48,6 @@ describe('ArrayElement', function() {
       });
     });
 
-    describe('#toValue', function() {
-      it('returns the array', function() {
-        expect(arrayElement.toValue()).to.deep.equal(['a', true, null, 1]);
-      });
-    });
-
```

Comment:
Please don't remove this test since the other tests you have written doesn't use `ArrayElement` to instantiate.